installer/config.yaml

```
@@ -1,27 +1,45 @@
+    - name: Trusted Artifact Signer
+      enabled: true
+      namespace: tssc-tas
       properties:
         manageSubscription: true
-    developerHub:
-      enabled: &rhdhEnabled true
-      namespace: &rhdhNamespace tssc-dh
+    # Red Hat Developer Hub is an enterprise-grade internal developer portal built
+    # on Backstage, designed to enhance developer productivity, collaboration, and
+    # onboarding by centralizing tools, documentation, and resources within a
+    # unified and extensible platform. 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)